examples/conf.d/daemon.conf

```diff
@@ -2,6 +2,9 @@
 #
```

Comment:
Can you split this change to another pr, so we can merge it now?

Reply:
Sure! Makes sense.